src/Waypoint/WaypointGlue.hpp

```diff
@@ -93,7 +93,8 @@ LoadWaypoints(Waypoints &way_points,
               ProgressListener &progress);
 
 /**
- * Append one waypoint to the file "user.cup".
+ * Recreate the file "user.cup", putting in it each waypoint in
```

Comment:
Can you improve this sentence? Its hard to understand.

Reply:
I'll try. :-) How's this...?

_Create the file "user.cup" (replacing "user.cup" if it already exists), writing to this new "user.cup" every waypoint in way_points with an "origin" of "USER" (i.e., every waypoint in memory that's marked as belonging in "user.cup")._

If it's still too hard to understand, please help point me to the hard-to-understand part, and I'll try again. (Once we have it like we want, I'll redo the commit, of course.)

Thanks,

Eric

Reply:
Can you update your pr? Then we can close this.